File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int left = 0;
        int right = numbers.length-1;
        while (left < right){
            int sumvalue = numbers[left]+numbers[right];
            if (sumvalue ==target){
                return new int[]{left+1, right+1};
            }
            else if(sumvalue < target){
                left++;}
            else{
                right--;}
        }
        return new int[]{-1,-1};
    }
}

File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        n = len(numbers)
        left = 0
        right = n-1
        while left < right:
            sumvalue = numbers[left] + numbers[right]
            if sumvalue==target:
                return [left+1, right+1]
            elif sumvalue<target:
                left+=1
            else:
                right-=1